docs/resources/Guild_Scheduled_Event.md

```diff
@@ -0,0 +1,136 @@
+| ---------------- | ------- | ----------------------------- |
+| with_user_count? | boolean | users subscribed to the event |
+
+## Create Guild Scheduled Event % POST /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/scheduled-events
+
+Create a guild scheduled event in the guild. Returns a [guild scheduled event](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object) object on success.
+
+###### JSON Params
+
+| Field                | Type                                                                                                                        | Description                                                                     |
+| -------------------- | --------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------- |
+| channel_id?          | snowflake                                                                                                                   | the channel id of the scheduled event, if for a stage instance of voice channel |
+| entity_metadata?     | [entity metadata](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-metadata) | the entity metadata of the scheduled event                                      |
+| name                 | string                                                                                                                      | the name of the scheduled event                                                 |
+| privacy_level        | [privacy level](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-privacy-level)     | the privacy level of the scheduled event                                        |
+| scheduled_start_time | ISO8601 timestamp                                                                                                           | the time to schedule the scheduled event                                        |
+| scheduled_end_time?  | ISO8601 timestamp                                                                                                           | the time when the scheduled event is scheduled to end                           |
+| description?         | string                                                                                                                      | the description of the scheduled event                                          |
+| entity_type          | [entity type](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-types)        | the entity type of the scheduled event                                          |
+
+## Get Guild Scheduled Event % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/scheduled-events/{guild_scheduled_event.id#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object}
```

Comment:
Does this not support the `with_user_count` query param like the List endpoint?

Reply:
`with_user_count` query param is not supported by this endpoint currently but we can look into getting this added so we stay consistent with the other endpoint. Good call out.